Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -595,6 +581,26 @@ public void Exit()
         }
 
         /// <summary>
+        /// Handles a generic order request
+        /// </summary>
+        /// <param name="request"><see cref="OrderRequest"/> to be handled</param>
+        /// <returns><see cref="OrderResponse"/> for request</returns>
+        internal OrderResponse HandleOrderRequest(OrderRequest request)
```

Comment:
I think this is a great idea, but we should make it `public` and have it do the complete handling of the request. This will effectively change the `Run` method such that it's main responsibility is just consuming the queue and invoking this method. Then we can call this method directly from tests knowing that it handles all processing of an `OrderRequest`